src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -298,7 +277,40 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
                 )}
               </ul>
             </nav>
-            {release.relatedInformation.length !== 0 && (
+            {release.publication.methodologies.length > 0 && (
+              <>
+                <h2
```

Comment:
h3 here as well? And for related pages heading below?

Reply:
Spoke with Marv and confirmed that indeed they should both be h3s as you suggested, so changed them both, ta!